From 3b7574de68d2479c63a13dafc9f45d0d12d39a83 Mon Sep 17 00:00:00 2001
From: Zebediah Figura <zfigura@codeweavers.com>
Date: Wed, 24 Jul 2019 14:44:15 -0500
Subject: [PATCH 0530/1533] server: Fix an invalid use of fsync_clear().

The fd object has no get_fsync_idx callback, so this would do nothing.
---
 server/fd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/fd.c b/server/fd.c
index c85d25e9566..a5ab3c4c46c 100644
--- a/server/fd.c
+++ b/server/fd.c
@@ -2172,7 +2172,7 @@ void set_fd_signaled( struct fd *fd, int signaled )
     if (signaled) wake_up( fd->user, 0 );
 
     if (do_fsync() && !signaled)
-        fsync_clear( &fd->obj );
+        fsync_clear( fd->user );
 
     if (do_esync() && !signaled)
         esync_clear( fd->esync_fd );
-- 
2.45.0

